src/typings/index.ts

```diff
@@ -43,6 +44,12 @@ export interface DataSetProps {
     collections?: any[];
 }
 
+// variation of react-apollo's ChildProps, with the props not marked as optional
+export type ChildProps<P, R> = P & {
```

Comment:
This is nice, but only applies to the ones that actually have a mutation. If there's no mutation this is not right.
For example: the search.tsx imports this, but does not have a mutate prop, because there's no mutation in place

Reply:
Sure, I'm fine with marking mutate as optional, I only need data to be 'unoptional' (if that's a term). I just thought mutate was always added (like data)